entries/e/emich.edu.json

```diff
@@ -6,6 +6,9 @@
       "custom-software",
       "custom-hardware"
     ],
+    "custom-software": [
+      "Duo"
+    ],
     "documentation": "https://www.emich.edu/it/security/esp/duo.php",
     "notes": "Available for employees",
```

Comment:
Is this necessary? Because their site says the following:
> All EMU accounts (employees, current students, recent students and alumni) must enroll in and use Duo to access most EMU online services.